File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/checkout/Cnt.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.checkout;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Cnt {

    @SerializedName("pos")
    @Expose
    private Long pos;
    @SerializedName("name")
    @Expose
    private String name;

    public Long getPos() {
        return pos;
    }

    public void setPos(Long pos) {
        this.pos = pos;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/Cnt.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Cnt {

    @SerializedName("pos")
    @Expose
    var pos: Long? = null
    @SerializedName("name")
    @Expose
    var name: String? = null

}
